packages/storybook-docs/src/components/Use.tsx

```diff
@@ -1,41 +1,37 @@
 import type { HTMLAttributes, ReactElement, PropsWithChildren } from 'react';
-// eslint-disable-next-line @talend/import-depth
-import { IconNameWithSize } from '@talend/icons/dist/typeUtils';
```

Comment:
remove dependency to icons as icon depends on it (cycle deps)

Reply:
result

![](https://github.com/Talend/ui/assets/32456736/a9ed5a50-5494-46b3-8d71-d324dc8e17a1)